tests/cache/tests.py

```diff
@@ -239,14 +239,14 @@ def custom_key_func(key, key_prefix, version):
 }
 
 
-def caches_setting_for_tests(base=None, **params):
+def caches_setting_for_tests(base=None, exclude=(), **params):
```

Comment:
I guess we shouldn't have mutable defaults -- use the same pattern as `bases` does?